### PR TITLE
Add clarification around vague "this" references

### PR DESCRIPTION
**Pull request by cipherboy**
As pointed out by @aphorise internally, the reference to "this" in the TLS section of the docs was vague. Clarify it and several other locations with vague use of the word "this".

### CODE REVIEW

**Comment by cipherboy** [comment]
\o @taoism4504 I tagged you because I thought you're usually auto-assigned to review docs PRs but you weren't this time, so I figured I'd add you and you can remove yourself if you don't think this is relevant for you to review :-)

**Review by jasonodonnell — APPROVED** [review]

File: website/content/docs/configuration/listener/tcp.mdx [hunk 1]
```diff
@@ -127,22 +127,27 @@ default value in the `"/sys/config/ui"` [API endpoint](/api/system/config-ui).
...
 
-  ~> **Note**: Go only consults this list for TLSv1.2 and earlier; the order of
-  ciphers is not important. For this parameter to be effective, the
-  `tls_max_version` property must be set to `tls12` to prevent negotiation of
-  TLSv1.3, which is not recommended. See the [Go blog post][go-tls-blog] for
-  more information.
+  ~> **Note**: Go only consults the `tls_cipher_suites` list for TLSv1.2 and
+  earlier; the order of ciphers is not important. For this parameter to be
+  effective, the `tls_max_version` property must be set to `tls12` to prevent
+  negotiation of TLSv1.3, which is not recommended. FOr more information about
```
Comment (jbayer) [inline website/content/docs/configuration/listener/tcp.mdx:141]: Typo here, FOr should be For

**Review by taoism4504 — COMMENTED** [review]
LG

File: website/content/docs/configuration/listener/tcp.mdx (hunk 1 above)
Comment (cipherboy) [reply website/content/docs/configuration/listener/tcp.mdx]: Ah good to hear :-) Thank you!

**Comment by cipherboy** [comment]
Thanks all for the reviews! Merging... :-)

**Pull request merged**